dashboard/web/flutter_bootstrap.js

```diff
@@ -0,0 +1,9 @@
+{{flutter_js}}
+{{flutter_build_config}}
+
+_flutter.loader.load({
+  onEntrypointLoaded: async function(engineInitializer) {
+    let appRunner = await engineInitializer.initializeEngine();
```

Comment:
What's this `engineInitializer` thingy?

Reply:
The object that gives you this:

* https://docs.flutter.dev/platform-integration/web/initialization#initializing-the-engine

Reply:
(In this case, it's totally optional since there's no customization of the bootstrap at all, but I wanted to preserve the chunk of code from the old `index.html` -> `flutter_bootstrap.js` without changes)